### PR TITLE
Added Matrix To Discourse plugin

### PR DESCRIPTION
**Pull request by gitayam**
A plugin create Discourse forum post from messages in Matrix or Bridged rooms and perform advanced forum searches directly from Matrix or Bridged rooms. Perfect for community building and engagement.

Functional with Matrix Rooms and Bridged Chats on Signal and WhatsApp

### CODE REVIEW

**Comment by gitayam** [comment]
A plugin create Discourse forum post from messages in Matrix or Bridged rooms and perform advanced forum searches directly from Matrix or Bridged rooms. Perfect for community building and engagement.



Added new line at the end for linter

File: data/plugins/thirdparty/matrix-to-discourse.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: Matrix To Discourse
+repo: https://github.com/gitayam/matrix-to-discourse
+license: GPL 3.0
```
Comment (tulir) [inline data/plugins/thirdparty/matrix-to-discourse.yaml:3]: The license field both here and in `maubot.yaml` should be a valid SPDX license identifier (https://spdx.org/licenses/)

```suggestion
license: GPL-3.0-or-later
```

or

```suggestion
license: GPL-3.0-only
```

Comment (gitayam) [reply data/plugins/thirdparty/matrix-to-discourse.yaml]: fixed it!

**Comment by gitayam** [comment]
A plugin create Discourse forum post from messages in Matrix or Bridged rooms and perform advanced forum searches directly from Matrix or Bridged rooms. Perfect for community building and engagement.

Added new line at the end for linter

**Comment by gitayam** [comment]
closing to open since it didn't pass the test the firsst time with the code. testing again with first time pass

**Pull request closed without merging**